src/config/dynamic-theme-fixes.config

```diff
@@ -2211,6 +2211,7 @@ my.nextdns.io
 
 INVERT
 img[src="/static/media/logo-large@2x.ac61cadf.png"]
+img[src^="data:image/png;"]
```

Comment:
This a pretty global selector doesn't this mess up other elements?

Reply:
No, it doesn't